.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -124,6 +124,7 @@
       <option name="m_ignoreLoopsWithoutConditions" value="false" />
     </inspection_tool>
     <inspection_tool class="ForLoopThatDoesntUseLoopVariable" enabled="true" level="WARNING" enabled_by_default="true" />
+    <inspection_tool class="FunctionName" enabled="false" level="WEAK WARNING" enabled_by_default="false" />
```

Comment:
Which inspection is this?  There doesn't seem to be a way to map between inspections' formal and informal names.

Reply:
It didn't like `o_*` names. I disabled the inspection, since it totally doesn't work for the project.